tests/components/sound.test.js

```diff
@@ -75,4 +71,19 @@ suite('sound', function () {
       assert.ok(sound.pause.called);
     });
   });
+
+  suite('play', function () {
```

Comment:
could test the sound component if used with multi, that the object3dname is set correctly